src/WebJobs.Script/runtimeassemblies.json

```diff
@@ -1,36 +1,7 @@
 ﻿// Assemblies in this list cannot be overriden in the funcion load context
 // This list originally comes from dumping TPA assemblies. 
-// In addition to what is on this list, all System.* assemblies are restricted.
```

Comment:
Also removed this comment as this is no longer the case. We expect all assemblies to be explicitly listed here.